src/reflect/scala/reflect/internal/Definitions.scala

```diff
@@ -435,7 +435,8 @@ trait Definitions extends api.StandardDefinitions {
     def elementType(container: Symbol, tp: Type): Type       = elementExtract(container, tp)
 
     // collections classes
-    private[this] lazy val isNewCollections = getClassIfDefined("scala.collection.IterableOnce") != NoSymbol
+    private[this] lazy val _isNewCollections = getClassIfDefined("scala.collection.IterableOnce") != NoSymbol
+    private[scala] def isNewCollections = _isNewCollections
```

Comment:
why not `private[scala] lazy val ...`? Or just make a public lazy val, we don't need to worry about visibility in reflect.internal.

Reply:
A non-`private[this]` val leaks into `JavaUniverseForce`